app/src/main/java/com/example/plantsapp/presentation/ui/plantcreation/SaveUriInStorageUseCase.kt

```diff
@@ -0,0 +1,56 @@
+package com.example.plantsapp.presentation.ui.plantcreation
```

Comment:
Also 2 points to mention: 
1. I think this is more like a data class - so put it into data layer
2. You can add interface to domain and use this interface in presentation
Also add package for usecase in domain and data